api/server/server_test.go

```diff
@@ -1,3 +1,6 @@
+// +build full_stack
+// +build server
+
```

Comment:
@kunihiko-t 

for example, like this,

> According to this document, multiple lines represent AND condition
> https://golang.org/pkg/go/build/#hdr-Build_Constraints

It is easier to follow context this way.

Thank you !